### PR TITLE
fix: Web 1258 token selector remove import from token list functionality

### PR DESCRIPTION
**Pull request by lynnshaoyu**
before:
d4a5c259-7437-4333-9eaa-bf45b2ee5371

after change:
Screen Shot 2022-09-26 at 7 24 30 PM

### CODE REVIEW

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 1]
```diff
@@ -137,8 +137,11 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
 
   // first price point on the x-axis of the current time period's chart
   const startingPrice = prices?.[0] ?? DATA_EMPTY
+  console.log('starting time', new Date(startingPrice.timestamp.valueOf() * 1000).toLocaleDateString(locale))
```
Comment (vm) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:140]: rm?

File: src/components/SearchModal/CurrencySearch.tsx [hunk 2]
```diff
@@ -26,8 +26,8 @@ import { isAddress } from '../../utils'
 import Column from '../Column'
 import Row, { RowBetween, RowFixed } from '../Row'
 import CommonBases from './CommonBases'
+import { CurrencyRow, formatAnalyticsEventProperties } from './CurrencyList'
 import CurrencyList from './CurrencyList'
-import ImportRow from './ImportRow'
```
Comment (vm) [inline src/components/SearchModal/CurrencySearch.tsx:30]: can this file be deleted?

File: src/components/SearchModal/CurrencySearch.tsx [hunk 3]
```diff
@@ -230,7 +230,20 @@ export function CurrencySearch({
...
         {searchToken && !searchTokenIsAdded ? (
           <Column style={{ padding: '20px 0', height: '100%' }}>
-            <ImportRow token={searchToken} showImportView={showImportView} setImportToken={setImportToken} />
+            <CurrencyRow
+              currency={searchToken}
+              isSelected={Boolean(searchToken && selectedCurrency && selectedCurrency.equals(searchToken))}
+              onSelect={(hasWarning: boolean) => searchToken && handleCurrencySelect(searchToken, hasWarning)}
+              otherSelected={Boolean(searchToken && otherSelectedCurrency && otherSelectedCurrency.equals(searchToken))}
+              showCurrencyAmount={showCurrencyAmount}
+              eventProperties={formatAnalyticsEventProperties(
```
Comment (vm) [inline src/components/SearchModal/CurrencySearch.tsx:239]: are we supposed to track for both of these?

Comment (lynnshaoyu) [reply src/components/SearchModal/CurrencySearch.tsx:239]: Yeah, we want to track when any token is selected.

File: src/components/SearchModal/CurrencySearch.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/components/SearchModal/CurrencySearch.tsx:30]: yeah i think it already has been.

**Review by vm — APPROVED** [review]

**Pull request merged**